README.md

```diff
@@ -204,6 +202,8 @@ this.http.get(url).subscribe(response => { /* do work */ });
 
 ## Extensibility endpoints
 
+> IMPORTANT - Please note that there is a known limitation, you cannot use the Angular binding syntax {{item.data.CreatedBy}}, when creating component HTML templates, you must use another binding, for example [textContent]="item.data.CreatedBy", or [innerHtml]="item.data.Content".
```

Comment:
Can we double check whether this is true? I suppose it was because the CommonModule was not provided and there is actually no problem

Reply:
it's because of the AOT, iris is built AOT the extensions are not

Reply:
I believe I recently tested this and it was working OK